lib/raven/configuration.rb

```diff
@@ -127,6 +127,12 @@ def initialize
       self.sanitize_fields = []
       self.sanitize_credit_cards = true
       self.environments = []
+
+      self.release = ENV['HEROKU_SLUG_COMMIT']
+
+      if self.release.nil? || self.release.empty?
+        self.release = `git rev-parse --short HEAD`.strip rescue nil
```

Comment:
Good call on `git` not existing on the system. I wrapped the sys call in a rescue: this will fail gracefully if the user doesn't have git installed, or if the current directory is not a git repo.